onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -331,11 +333,13 @@ static void DumpOpenVINOEPModel(const std::filesystem::path& onnx_model_path_nam
     if (dash != std::string::npos) {
       auto new_name = model_name.stem().string() + subgraph_name.substr(dash, std::string::npos);
       model_name.replace_filename(new_name);
+      model_name.replace_extension(".onnx");
```

Comment:
Isn't the input model a .onnx file?

Reply:
This is a debug model we dump for reference after the ORT optimizations